pxd.c

```diff
@@ -238,6 +245,107 @@ static long pxd_ioctl_run_user_queue(struct file *file)
 	return 0;
 }
 
+static void print_io_flusher_state(unsigned int new_flags,
+				   pid_t pid, pid_t ppid, char *comm)
+{
+	if (IS_SET_IO_FLUSHER_FLAG(new_flags)) {
+		printk("Process %s pid %d parent pid %d IO_FLUSHER is set\n",
+			       comm, pid, ppid);
+	} else {
+		printk("Process %s pid %d parent pid %d IO_FLUSHER not set\n", comm, pid,
+				ppid);
+	}
+}
+
+static int is_io_flusher_supported(void)
+{
+	#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0) && LINUX_VERSION_CODE <= KERNEL_VERSION(5,8,0)
+		#if defined(PF_MEMALLOC_NOIO) && defined(PF_LESS_THROTTLE)
+			return 1;
+		#else
+			return 0;
+		#endif
+	#else
+		return 0;
+	#endif
+}
+
+static long pxd_ioflusher_state(void __user *argp)
+{
+	struct pxd_ioctl_io_flusher_args io_flusher_args;
+	struct task_struct *task = NULL;
+	struct pid_namespace *ns = NULL;
+	pid_t ppid = -1;
+	pid_t pid = -1;
+	char *comm = NULL;
+	unsigned int old_flags = 0;
+	unsigned int new_flags = 0;
+	char command[TASK_COMM_LEN] = {0};
+	int is_io_flusher_set = 0;
+
+	if (!is_io_flusher_supported()) {
+		return -EOPNOTSUPP;
+	}
+
+	if (argp == NULL)
```

Comment:
nit: braces to be uniform with other ifs

Reply:
fixed it